### DISCUSSION TITLE
Max export size error

### DISCUSSION THREAD

**Discussion by zilphy** [outline/outline#7727] [Self Hosting] [answered]
Hello! I am running into a wall to get my Outline instance exported. I'm trying to move from local storage to S3 on my instance. I assume I can export, change the storage location and re-import. I'm not even sure if that would work... I feel like I might need to start over to get S3 working. Either way I receive the following error when trying to export all collections. When I check the server logs, I get the same message with no additional information.

![image](https://github.com/user-attachments/assets/4bf7ff70-7ffc-4cc4-977f-8da53db7a5f2)

Is there a way to override this behavior?

**Comment by tommoor** [comment, marked as answer]
The export is bigger than the available memory, you need to increase the memory available to the server / container
